### PR TITLE
Initial Push

### PR DESCRIPTION
**Pull request by prak24oct**

### CODE REVIEW

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]
rename the java files

File: index.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import { 
+    NativeEventEmitter,
+    NativeModules,
+    Platform,
+} from 'react-native';
+
+const module_trackier = NativeModules.TrackierSDK;
+
+let module_adjust_emitter = null;
```
Comment (Hemant-Mann) [inline index.js:11]:
```suggestion
let module_trackier_emitter = null;
```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+if (Platform.OS === "android") {
+    module_adjust_emitter = new NativeEventEmitter(NativeModules.TrackierSDK);
```
Comment (Hemant-Mann) [inline index.js:13]:
```suggestion
    module_trackier_emitter = new NativeEventEmitter(NativeModules.TrackierSDK);
```

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/com/trackier/reactnsdk/TrackierSDK.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        Map<String, Object> ev = TrackierUtil.toMap(trackierEventMap.getMap("ev"));
+        if (null != ev) {
+            for (Map.Entry<String, Object> entry : ev.entrySet()) {
+                addEventValues(entry.getKey(), entry.getValue().toString());
+            }
+        }
+        com.trackier.sdk.TrackierSDK.trackEvent(trackierEvent);
+    }
+
+    public void addEventValues(String key, String value) {
```
[the remark this answers is not in the document]
Comment (prak24oct) [reply android/app/src/main/java/com/trackier/reactnsdk/TrackierSDK.java:55]: This function is making haspMap for eventValues.

**Review by Hemant-Mann — APPROVED** [review]

**Pull request merged**